updating some links and link text in the readme (#442)
pulumi/pulumi-aws-native: README.md

## Configuring credentials

> Credentials configuration is compatible with the classic AWS provider. Refer to the [AWS configuration options](

Reviewer: Same note as in Azure-Native - the compatibility claim is somewhat loaded, I'd be happy to remove it.

> Credentials configuration is compatible with the classic AWS provider. Refer to the [AWS configuration options](
> https://www.pulumi.com/registry/packages/aws-native/installation-configuration/#configuration-options).

Author: ```suggestion
To learn how to configure credentials refer to the [AWS configuration options](https://www.pulumi.com/registry/packages/aws-native/installation-configuration/#configuration-options).
```
